L15/index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>L15</title>
+</head>
+<style type="text/css">
+
+		.galleryBox {
+			display: block;
+			border: 1px;
+			border-style: solid;
+			overflow: hidden;
+			width: 600px;
+			height: 400px;
+			position: relative;
+			margin: auto;
+			object-fit: cover;
+		}
+
+		.image {
+			display: none;
+			height: 100%;
+			width: 100%;
+			}
+
+		.active {
+			display: block;
+		}
+
+		.prev, .next {
+			cursor: pointer;
+			position: absolute;
+			top: 50%;
+			width: auto;
+			padding: 16px;
+			margin-top: -22px;
+			color: white;
+			font-weight: bold;
+			font-size: 18px;
+			transition: 0.6s ease;
+			border-radius: 0 3px 3px 0;
+		}
+
+		.next {
+			right: 0;
+		}
+
+	</style>
+<body>
+	<h2>Gallerey</h2>
+	<p>
+		Write html page with gallery (you can also call carousel) no javascript frameworks or libraries. The only requirement for the gallery is for it to work and want to be remotely about galleries that you can find on the Internet.<br/><br/>
+		P.S. The recommendation is that the first one does not aggravate the “triggering” events. Quite plain enough with buttons, BUT personally written.
+	</p>
+	<div class="gallery">
+		<div class="galleryBox"; id="galleryBox1"></div>
+		<div class="galleryBox"; id="galleryBox2"></div>
+	</div>
+</body>
+<script type="text/javascript">
+
+	var arrWithFoto1 =[
+	"images/image11.jpg",
+	"images/image22.jpg",
+	"images/image33.jpg",
+	"images/image44.jpg",
+	"images/image55.jpg"
+	], array = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20], idBox1 = "galleryBox1", idBox2 = "galleryBox2";
+
+
+	function Gallery (arrWithSrcFoto, idBox){
+		this.arrWithImgSrc = arrWithSrcFoto;
+		this.containerId = idBox;
+		this.arrElement = this.genArrayWhithElement(), this.setFirstActive(), this.runButtons();
```

Comment:
I am recommend to call function from new line. And avoiding to use multi assign syntax for that.